src/Tribe/Tickets.php

```diff
@@ -2838,7 +2838,7 @@ protected function update_capacity( $ticket, $data, $save_type ) {
 			}
 
 			// Fetch the stock if defined, otherwise use Capacity field
-			$data['stock'] = trim( Tribe__Utils__Array::get( $data, 'stock', $data['capacity'] ) );
+			$data['stock'] = (int) trim( Tribe__Utils__Array::get( $data, 'stock', $data['capacity'] ) );
```

Comment:
Not needed for this fix but also the conditional below needs it to be set as a string

```suggestion
			$data['stock'] = trim( Tribe__Utils__Array::get( $data, 'stock', $data['capacity'] ) );
```